src/backend/XmlHttpRequest.ts

```diff
@@ -31,6 +33,10 @@ export interface XmlHttpRequestOptions {
   // Used as the URL prefix for fetched files.
   // Default: Fetch files relative to `url`.
   baseUrl?: string;
+  // Whether to use Fetch or XmlHttpRequest.
+  // Can be 'xhr', 'fetch', or 'auto'.
+  // Default: 'auto' which uses fetch if its available, and XHR as a fallback
```

Comment:
(I actually would like to support fetch's streaming downloads at some point to avoid prefetching entire files before any operations can proceed, but that's a later PR.)

I'll think on this some more.